cmake-maven-plugin/src/main/java/com/googlecode/cmakemavenproject/CompileMojo.java

```diff
@@ -125,12 +125,12 @@ public void execute()
 
 			// We assume that someone else has extracted the cmake binaries beforehand (e.g. the
 			// "generate" mojo).
-			File cmakeFile = downloadBinaries ? new File(project.getBuild().getDirectory(),
-				"dependency/cmake/bin/cmake") : new File(cmakeRootDir + "/" + cmakeChildDir);
+			String cmakePath = downloadBinaries ? new File(project.getBuild().getDirectory(),
+				"dependency/cmake/bin/cmake").getAbsolutePath() : cmakeRootDir + cmakeChildDir;
```

Comment:
To work on windows, this should be
String cmakePath = new File(downloadBinaries ? project.getBuild().getDirectory(),
				"dependency/cmake") : cmakeRootDir).getAbsolutePath() + cmakeChildDir;